src/sketch.js

```diff
@@ -1,21 +1,28 @@
 /**
  * p5.js setup.
- * 
+ *
  * Returns:
  *  undefined.
  */
-function setup() 
-{
+let song;
 
+function preload() {
```

Comment:
Please document this function.

```suggestion
/**
* p5.js preload callback.
*
* Returns:
*   undefined;
*/
function preload() {
```